ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -349,6 +353,33 @@ - (void)_showOrReconfigureManagedAppSplashScreen:(NSDictionary *)manifest
   }
 }
 
+- (nullable NSString *)_loadingViewTextForStatus:(EXAppLoaderRemoteUpdateStatus)status
+{
+  if (status == kEXAppLoaderRemoteUpdateStatusChecking) {
+    return @"Checking for new release...";
+  } else if (status == kEXAppLoaderRemoteUpdateStatusDownloading) {
+    return @"New release available, downloading...";
+  } else {
+    return nil;
+  }
+}
```

Comment:
I would move this to the `EXAppLoadingProgressWindowController` class to keep this UIViewController as clean and small as possible - I'm trying to stick more to SOLID idea and other nicely shaped OOP guidelines and techniques 😊 <I hope this would make our codebase a little less confusing and easier to maintain>